encrypted-media/drm-keystatuses-multiple-sessions.html

```diff
@@ -39,11 +39,12 @@
         .then( function( contents ) {
 
             // Select a content item with multiple keys and initData
-            var contentitem = contents.filter( function( item ) { return item.keys.length > 1 && item.initDataType && !item.associatedInitData; } )[ 0 ];
+            var contentitem = contents.filter( function( item ) { return item.keys.length > 1 && item.initDataType && !item.associatedInitData; } )[ 0 ],
```

Comment:
Ah, I didn't process that the next line was red. Personally, I find multiline var statements harder to read (as demonstrated here), but I'll defer to WPT style. Also, this would be a general cleanup.